pennylane_lightning_gpu/src/algorithms/AdjointDiffGPU.hpp

```diff
@@ -550,17 +816,17 @@ template <class T = double> class AdjointJacobianGPU {
      * @param ref_data Pointer to the statevector data.
      * @param length Length of the statevector data.
      * @param jac Preallocated vector for Jacobian data results.
-     * @param obs Observables for which to calculate Jacobian.
+     * @param obs ObservableGPUs for which to calculate Jacobian.
      * @param ops Operations used to create given state.
      * @param trainableParams List of parameters participating in Jacobian
      * calculation.
      * @param apply_operations Indicate whether to apply operations to psi prior
      * to calculation.
      */
+
     void batchAdjointJacobian(
-        const CFP_t *ref_data, std::size_t length,
-        std::vector<std::vector<T>> &jac,
-        const std::vector<Pennylane::Algorithms::ObsDatum<T>> &obs,
+        const CFP_t *ref_data, std::size_t length, std::vector<T> &jac,
```

Comment:
The Jacobian array is now passed in as 1D and reshaped on the other side?

Reply:
yes!